packages/cli/src/commands/publish.ts

```diff
@@ -48,14 +48,35 @@ export default class Publish extends Command {
     } else {
       throw new Error('Please provide SUBQL_ACCESS_TOKEN before publish');
     }
-    const cid = await uploadToIpfs(project.manifest, authToken.trim(), flags.ipfs).catch((e) => this.error(e));
-    await createIPFSFile(location, cid);
-
-    if (!flags.output) {
-      this.log('Uploading SubQuery project to IPFS');
-      this.log(`SubQuery Project uploaded to IPFS: ${cid}`);
-    } else {
-      this.log(`${cid}`);
+    let directory: string;
+    if (project.manifests.length > 1) {
+      directory = path.basename(project.root); // Using project root name as directory
+      // or use a timestamp for uniqueness
+      directory += `_${new Date().getTime()}`;
     }
```

Comment:
I don't think we need this directory stuff. Its not relevant and just makes the logging confusing